process/pretty_print.go

```diff
@@ -104,26 +107,26 @@ func (p PrettyPrint) ApplyBatch(ctx context.Context, caps []config.Capsule) ([]c
 	var count int
 	var stack []byte
 
-	newCaps := newBatch(&caps)
-	for _, cap := range caps {
-		ok, err := op.Operate(ctx, cap)
+	newCaps := newBatch(&capsules)
```

Comment:
similar to the other comment, is `newCapsules` preferred since we're changing the convention to `capsule`?